docs/src/conversions.md

```diff
@@ -74,6 +74,20 @@ r = robject(d)
 rcopy(r)
 ```
 
+Note that R's `POSIXct` supports higher precision than DateTime:
+
+```@example 1
+r = reval("as.POSIXct('2020-10-09 12:09:46.1234')")
+```
+
+```@example 1
+d = rcopy(r)
+```
+
+!!! note "Conversions to `DateTime` are given in UTC!"
+    `POSIXct` stores times internally as UTC with a timezone attribute.
+    The conversion to `DateTime` necessarily strips away timezone information, resulting in UTC values.
```

Comment:
If `POSIXct` has a notion of timezone, I wonder if it should actually be represented on the Julia side as a `ZonedDateTime` rather than a `DateTime`. That'd be a breaking change but something possibly worth considering, at least if I understand the situation correctly.

Reply:
yep, hence #535!